introducing-polyglotweekly.com.md

```diff
@@ -0,0 +1,193 @@
+<!--
+headline: Introducing polyglotweekly.com
+description: introducing the Polyglot Weekly project
+author: Ben Coe
+datePublished: 2015-04-05
+twitter: benjamincoe
+github: bcoe
+-->
+
+It's been almost a year since I wrote an article about programming, this makes me me sad.
+
+> Polyglot Weekly grows from the question, "what would get me blogging more often?"
+
+I love writing. It forces me to step back, reflect on what I've been learning, and helps solidify it.
+
+Polyglot Weekly grows from the question, "what would get me blogging more often?" I came up with a list of answers:
+
+* It would be fun to write as part of a community. I love the idea of
+  [The Pastry Box](https://twitter.com/thepastrybox), which compiles a curated list of articles about the web. Wouldn't it be neat if there was a similar initiative for articles about programming?
+* I wanted a platform that rendered source-code in a beautiful way,
+  other blogging platforms I've used have been so so at this.
+* I'd love to read articles from a variety programming communities.
+  We can all learn from each other's methodologies, and it's important to discourage  inter-community shade throwing.
+* I love working with an editor. As frustrating as it can sometimes be, it helps
+  ensure that I write quality articles. I wanted to create a platform that
+  had an editorial process.
+
+## How Polyglot Weekly Works
+
+### Generating an Article
+
+Anyone can submit a pull request for an article to Polyglot Weekly, simply run:
+
+```shell
+git clone https://github.com/polyglotweekly/polyglotweekly.com.git
+cd polyglotweekly.com
+nvm use 0.10 # sorry, no 0.12.x or iojs yet.
+npm install
+npm run generate
+```
+
+Keep the following contribution guidelines in mind:
+
+* Articles should be on the topic of programming: articles advertising your start-up,
+  discussing Bay-Area culture, or pointing out your favorite dog grooming techniques will be politely declined.
+* keep contributions positive! your article should not be a rant about why
+  Ruby is better than Go. Feel free to criticize aspects of your language
+  of expertise, but do so in a constructive manner.
+* Provide concrete examples in code. Polyglot Weekly uses a [great syntax-highlighter](https://github.com/atom/highlights) extracted from the Atom Text Editor, take advantage of it.
+
+### The Editorial Process
+
+Articles are submitted to the [polyglotweekly.com repository](https://github.com/polyglotweekly/polyglotweekly.com) as a pull request. This allows a conversation to take place with the community surrounding the article. This discussion may include:
+
+* grammatical fixes and other editorial nits.
+* suggestions about structure, e.g., adding a section to an article that might
+  pull the whole piece together.
+* as with the tone of articles, the goal will be to keep the editorial process positive.
```

Comment:
If there are any suggested guidelines around the structure of PRs, they should probably be here *or pointed to here*. An example would be branch naming.